src/test/java/uk/gov/hmcts/reform/slc/services/UpdateLetterStatusJobTest.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.slc.services;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.slc.services.steps.sftpupload.FtpClient;
+
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class UpdateLetterStatusJobTest {
```

Comment:
Test class name says `UpdateLetterstatus` but test methods are testing reports can be downloaded or not?

Reply:
`UpdateLetterStatusJob` is the name of the class that's tested

Reply:
`UpdateLetterStatusJob` downloading files does not sound correct to me tbh